packages/salesforcedx-vscode-core/package.json

```diff
@@ -807,6 +812,14 @@
           "dark": "resources/dark/retrieve.svg"
         }
       },
+      {
+        "command": "sfdx.force.source.retrieve.open.component",
+        "title": "%force_source_retrieve_display_text%",
```

Comment:
Let's add a new label for this button in `package.nls.json` that's something like "Retrieve and Open Source From Org" so that it's different from the other button which says "Retrieve Source from Org"

cc: @ekapner

Reply:
Retrieve and Open Source (I think the from org is implied unless someone disagrees)

Reply:
But I agree it should be different from other button text

Reply:
The other button adds "Org" at the end, but I think your suggestion makes sense

Reply:
Done! Using this wording: "Retrieve and Open Source"